index.d.ts

```diff
@@ -98,6 +99,27 @@ export interface Space {
     toPlainObject(): Space;
 }
 
+export interface Locales {
```

Comment:
Sorry for the confusion, we don't need to have ´Locales´ Type since we have ContentfulCollection.
The type definition for locales should be like the following.

```js
getLocales(): Promise<LocaleCollection>;

// ....

export interface Locale {
  code: string
  name: string
  default: boolean
  fallbackCode: string | null
  sys: {
    id: string
    type: 'Locale'
    version: number
  }
}

export type LocaleCollection = ContentfulCollection<Locale>;
```